.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "brightspace/node-config",
```

Comment:
I don't know the differences between all the linting configs offhand - I'm guessing the Lit one doesn't give us anything extra if there's no Lit components?